tests/Examples/swift-bindings/core/basic.txt

```diff
@@ -0,0 +1,7 @@
+# Check basic 'core' functionality of the swift bindings
+#
+# RUN: xcrun -sdk macosx swiftc %{srcroot}/examples/swift-bindings/core/basic.swift -I %{srcroot}/build/products/swift-bindings -L %{llbuild-lib-dir} -Xcc -fmodule-map-file=%{srcroot}/products/libllbuild/public-api/llbuild/module.modulemap -emit-executable %{srcroot}/build/products/swift-bindings/CMakeFiles/llbuild.swift.o -lllbuild -o %t.exe
```

Comment:
Done the second part, now a `swift-binding.dylib` is generated under `build/lib`
trying to figure out lit.